index.html

```diff
@@ -0,0 +1,246 @@
+        <link rel="stylesheet" href="./style/body/subscribe/__text/subscribe__text.css"/>
+        <link rel="stylesheet" href="./style/body/subscribe/subscribe-form/subscribe-form.css"/>
+        <link rel="stylesheet" href="./style/body/subscribe/subscribe-form/__input/subscribe-form__input.css"/>
+        <link rel="stylesheet" href="./style/body/models/models.css"/>
+        <link rel="stylesheet" href="./style/body/models/panel/panel.css"/>
+        <link rel="stylesheet" href="./style/body/models/panel/__content/panel__content.css"/>
+        <link rel="stylesheet" href="./style/body/footer/footer.css"/>
+        <link rel="stylesheet" href="./style/body/footer/__content/footer__content.css"/>
+        <link rel="stylesheet" href="./style/body/footer/__content/__text/footer__text.css"/>
+        <link rel="stylesheet" href="./style/body/footer/__content/link-button/link-button.css"/>
+    </head>
+    <body class="body">
+        <header class="header">
+            <a href="" class="logo">
+                <span class="text_size_big logo__name">WEBSITE-NAME</span>
+                <span class="text_size_small logo__slogan">Jevakiro puso od ia javai</span>
+            </a>
+            <ul class="menu">
+                <li><a class="menu__item home-button" href="">HOME</a></li>
+                <li><a class="menu__item" href="#about">ABOUT</a></li>
+                <li><a class="menu__item" href="#services">SERVICES</a></li>
+                <li><a class="menu__item" href="#portfolio">PORTFOLIO</a></li>
+                <li><a class="menu__item" href="#contact">CONTACT</a></li>
+            </ul>            
+        </header>
+        <div class="banner">
+            <div class="banner__img-with-text">
+                <div class="banner__filter banner__filter_position_top-left"></div>
+                <span class="banner__text banner__text_color_black">
+                    JEVAKIRO PUSO
+                </span>
+            </div>
+            <div class="banner__img-with-text">
+                <div class="banner__filter banner__filter_position_top-right"></div>
+                <span class="banner__text banner__text_color_black">
+                    JEVAKIRO PUSO
+                </span>
+            </div>
+            <div class="banner__img-with-text">
+                <div class="banner__filter banner__filter_position_bottom-left"></div>
+                <span class="banner__text banner__text_color_black">
+                    JEVAKIRO PUSO
+                </span>
+            </div>
+            <div class="banner__img-with-text">
+                <div class="banner__filter banner__filter_position_bottom-right"></div>
+                <span class="banner__text banner__text_color_orange">
+                    JEVAKIRO PUSO
+                </span>
+            </div>
+        </div>
+        <div class="title">
+            <span class="title__text text_size_big">
+                JEVAKIRO PUSO OD IA JAVAI I WILEKUD U SAXUHAF
```

Comment:
Some <span> tags have been replaced by <h*> in the header, title, row, models and footer sections. Besides it, I should reset default margins in my .css files for <h1> because they are too large.